### ISSUE TITLE
wrong abbreviation for month »February«

### ISSUE THREAD

**Issue by Edelweiss** [DCLP/dclpxsltbox#362]
HGV abbreviates »February« as »Febr.«

Therefore, on this line of code

https://github.com/DCLP/sosol/blob/master/app/helpers/hgv_meta_identifier_helper.rb#L1904

```Feb.``` should become ```Febr.```

```
months = ['', 'Jan.', 'Feb.', 'März', 'Apr.', 'Mai', 'Juni', 'Juli', 'Aug.', 'Sept.', 'Okt.', 'Nov.', 'Dez.']
```

**Comment by Edelweiss** [comment]
grep result for »Feb\.«

```
./HGV_meta_EpiDoc//HGV122/121935.xml:33: <origDate when="0485-02-17">17. Feb. 485</origDate>
./HGV_meta_EpiDoc//HGV44/43029.xml:78: <p>Alternativdatierung: 19. Jan. - 5. Feb. 193 v. Chr.</p>
./HGV_meta_EpiDoc//HGV46/45353.xml:34: <origDate notBefore="0710-01-07" notAfter="0710-02-05">7. Jan. - 5. Feb. 710</origDate>
```

**Comment by Edelweiss** [comment]
On papyri/idp.data (cherry-picked from DCLP https://github.com/DCLP/idp.data/tree/issue362)
https://github.com/papyri/idp.data/commit/c9de79c6b80af426a60dc9060302ec1b7b3536b2

SoSOL (https://github.com/DCLP/sosol/tree/issue362)
https://github.com/DCLP/sosol/commit/40fdb48a5c1b0b29a5023f3ebae8165c5e865013
(merged to development and master)

**Comment by ryanfb** [comment]
The tests in `date_test.rb` should also be updated to the new abbreviation (currently [this](https://github.com/DCLP/sosol/commit/40fdb48a5c1b0b29a5023f3ebae8165c5e865013) causes the tests to fail): https://github.com/DCLP/sosol/blob/master/test/unit/date_test.rb

**Comment by Edelweiss** [comment]
Yes, thank you for this report and sorry for the trouble. Updated tests accordingly in the following commit:

https://github.com/DCLP/sosol/commit/f8515f0dfd7dc09598d030b3d45ed5d01b99ee5d
(merged to development and master)

**Comment by ryanfb** [comment]
Thanks!
